examples/ValidationErrorMessage/ValidationErrorMessage.md

```diff
@@ -0,0 +1,5 @@
+```jsx
+<ValidationErrorMessage>
+  Error: This entry may only contain numbers.
+</ValidationErrorMessage>
+```
```

Comment:
It looks like this example isn't being rendered because the `example/` directory structure should match the `src/` directory structure.

Reply:
Ah ok I see!! Updated through 79ed1dc